app/controllers/addresses_controller.rb

```diff
@@ -0,0 +1,51 @@
+class AddressesController < ApplicationController
+    def index 
+        address = Address.all 
+        render json: address 
+    end
+
+    def show
+      id = params[:id]
+      address = Address.find(id)
+      render json: address
+    end
+
+    def new 
```

Comment:
Do we need this action? I think the form on the front end will take care of this.